pkg/commands/oscommands/os_default_platform.go

```diff
@@ -4,14 +4,23 @@
 package oscommands
 
 import (
+	"os"
 	"runtime"
 )
 
+func getShell() string {
+	defaultShell := "bash"
+	if shell := os.Getenv("SHELL"); shell != defaultShell {
+		return shell
+	}
+	return defaultShell
+}
+
 func GetPlatform() *Platform {
 	return &Platform{
 		OS:              runtime.GOOS,
-		Shell:           "bash",
-		ShellArg:        "-c",
+		Shell:           getShell(),
+		ShellArg:        "-ic",
```

Comment:
what does the i flag do and how many shells support it?

Reply:
It makes the shell `interactive` which in turn reads startup files (among other things), which allows you to use aliases, functions and variables defined in your e.g. `.zshrc`.

I tried it with `bash`, `zsh` and `fish` and all of them support the `i` flag.

Reply:
Wow very cool. I wonder if this will slow things down for users that have extensive e.g. zshrc files.